.github/workflows/process_opened_issues.yml

```
@@ -0,0 +1,17 @@
+name: Process (re)opened issues
+
+on:
+  issues:
+    types:
+      - reopened
+      - opened
+
+jobs:
+  add-to-project:
+    name: Adding issue to the master project
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/add-to-project@main
```

Use a specific commit hash: https://opensource.google/documentation/reference/github/services.md#gha-bestpractices

```suggestion
      - uses: actions/add-to-project@b56023be57e9c76f06e3844f18a1ab5dec7e7617
```